checks/microbenchmarks/cpu/alloc_speed/alloc_speed.py

```diff
@@ -42,7 +26,7 @@ def assert_4GB(self):
 
     @run_before('performance')
     def set_reference(self):
-        base_perf = 0.12
+        base_perf = 0.2
```

Comment:
since this is a cpu only test, let's try to follow the guidelines here ;-) 
```
'zen2:*': ... 
```
instead of:
```
'hohgant:nvgpu': 
```
?

together with
```
        self.skip_if_no_procinfo()
        processor_info = self.current_partition.processor
        processor_info.topology['arch']) ...
```
?